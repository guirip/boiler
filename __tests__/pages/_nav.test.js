import renderer from 'react-test-renderer'
import Nav from '../../pages/_nav'
import { v4 as uuid } from 'uuid'

it('Returns correct Nav when user logged out', async () => {
  const tree = renderer.create(<Nav user={null} />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('Returns correct Nav when user logged in', async () => {
  const user = {
    email: '123',
    id: uuid()
  }

  const tree = renderer.create(<Nav user={user} />).toJSON()
  expect(tree).toMatchSnapshot()
})
