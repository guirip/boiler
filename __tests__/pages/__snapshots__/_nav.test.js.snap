// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Returns correct Nav when user logged in 1`] = `
<div
  style={
    Object {
      "marginBottom": "20px",
    }
  }
>
  <a
    href="/dashboard"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    Dashboard
  </a>
   
  <a
    onClick={[Function]}
  >
    Logout
  </a>
</div>
`;

exports[`Returns correct Nav when user logged out 1`] = `
<div
  style={
    Object {
      "marginBottom": "20px",
    }
  }
>
  <a
    href="/login"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    Login
  </a>
   
  <a
    href="/"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    Home
  </a>
</div>
`;
